client/my-sites/earn/memberships/products.jsx

```diff
@@ -83,7 +110,10 @@ class MembershipsProductsSection extends Component {
 		editedMultiplePerUser: false,
 		editedPrice: { currency: 'USD', value: '' },
 		editedSchedule: '1 month',
+		editedPostsEmail: false,
+		editedCustomConfirmationMessage: '',
```

Comment:
Naming-wise should we maybe more closely match `welcome_email_content` and `subscribe_as_site_subscriber`.

So following patterns here maybe: `editedWelcomeEmailContent` and `editedSubscribeAsSiteSubscriber`

Reply:
I used names more close to the UI copy following the current pattern we have for other properties (i.e. [`editedPayWhatYouWant` vs `buyer_can_change_amount`](https://github.com/Automattic/wp-calypso/blob/8874e7fe95739378031f5514c5873b5981e7a843/client/my-sites/earn/memberships/products.jsx#L142) or [`editedSchedule` vs `interval`](https://github.com/Automattic/wp-calypso/blob/8874e7fe95739378031f5514c5873b5981e7a843/client/my-sites/earn/memberships/products.jsx#L141)).

But happy to rename them if folks agree we should move to a different pattern 👍